convert_frames_to_video.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+def convert_frames_to_video(input_path, output_path, fps):
```

Comment:
I guess we should probably also check whether `output_path` exists here? Because otherwise, `ffmpeg` will replace the video file without asking.